cpu/samd21/periph/adc.c

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (C) 2017 Dan Evans <photonthunder@gmail.com>
+ * Copyright (C) 2017 Travis Griggs <travisgriggs@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#include <stdint.h>
+#include "cpu.h"
+#include "periph/gpio.h"
+#include "periph/adc.h"
+#include "periph_conf.h"
+#include "mutex.h"
+
```

Comment:
fixed (I think I did what you wanted, found something similar in nrf51 variant). resquashed as well